### PR TITLE
ECE651-75 Order Now Button UI and Redirection

### PR DESCRIPTION
**Pull request by MareenaFr**
Order Now button style changed and redirection to restaurants listing page

Test results location - Documents/General/Sprint 4/Development/ECE651-75

### CODE REVIEW

**Review by kj8patel — APPROVED** [review]
LGTM!!
Thanks for making the changes.

File: src/components/OrderNow.js [hunk 1]
```diff
@@ -4,11 +4,13 @@ import { Link } from "react-router-dom";
...
   return (
     <div className="border" data-testid={dataTestId}>
-      <div className="nav-item text-white">
-        <Link to="/restaurants">
-          <button type="button" className="btn bg-orange h-12 px-6">
-            ORDER NOW
-          </button>
+      <div className="text-center py-5 home-cont3 text-white h-68 bg-no-repeat bg-order-now-background">
+          <p className="mt-20 h1 text-white text-3xl font-bold mt-5 mb-3">JUST ORDER AND WE WILL DELIVER YOU</p>
+          <p className="mb-3 text-white">Pellentesque eget justo eget nibh luctus semper at ut tellus.</p>
```
Comment (kar12thik) [inline src/components/OrderNow.js:9]: Can we put some actual text here ?

**Review by kar12thik — APPROVED** [review]
Changes look good.

**Review by db8patel — DISMISSED** [review]
It looks nice. Test results and code changes seems perfect.

**Review by itsmejay80 — CHANGES_REQUESTED** [review]
When I click on Order Now it is getting redirected to the restaurant's page this is correct but not at the top of the page. Refer this for solution  https://stackoverflow.com/questions/33188994/scroll-to-the-top-of-the-page-after-render-in-react-js

**Comment by MareenaFr** [comment]
> When I click on Order Now it is getting redirected to the restaurant's page this is correct but not at the top of the page. Refer this for solution https://stackoverflow.com/questions/33188994/scroll-to-the-top-of-the-page-after-render-in-react-js

Will check on this.

**Comment by MareenaFr** [comment]
> When I click on Order Now it is getting redirected to the restaurant's page this is correct but not at the top of the page. Refer this for solution https://stackoverflow.com/questions/33188994/scroll-to-the-top-of-the-page-after-render-in-react-js

Made the required changes.

**Review by itsmejay80 — APPROVED** [review]
LGTM

**Review by kj8patel — APPROVED** [review]

**Review by db8patel — APPROVED** [review]

**Review by Riya0001 — APPROVED** [review]
Looks good proceed with changes

**Review by kar12thik — APPROVED** [review]

**Pull request merged**